src/shim.js

```diff
@@ -132,7 +161,13 @@ function executeCommand(command) {
         const target = objectSpace[command.target];
         const method = target[command.name];
 
-        if (haveCallbacks.indexOf(command.name) === -1) {
+        if (command.name === 'addEvent') {
```

Comment:
Ugh, I just noticed something, shouldn't this block just be in `const commands = { ... }` block above instead. You wouldn't have to do all this if statements and just copy the body of `addEvent` to `commands.addEvent`. That would be much cleaner.